webapp/src/main/java/com/box/l10n/mojito/smartling/response/Response.java

```diff
@@ -7,7 +7,7 @@ public class Response<T> {
 
   String code;
   T data;
-  Errors errors;
+  Errors[] errors;
```

Comment:
Can we use List here instead of a typed array?

e.g. `List<Errors> errors`

Reply:
Done